crates/relayer/tests/integration.rs

```diff
@@ -31,9 +33,12 @@ async fn test_sync() {
     let source_db = node_server.conn_pool.clone();
 
     let mut test_conn = relayer_conn.acquire().await.unwrap();
-    let tx = test_conn.transaction().unwrap();
-    db::create_tables(&tx).unwrap();
-    tx.commit().unwrap();
+
+    node_db::with_tx::<_, QueryError>(&mut test_conn, |tx| {
+        db::create_tables(tx)?;
+        Ok(())
+    })
+    .unwrap();
```

Comment:
```suggestion
    node_db::with_tx(&mut test_conn, |tx| db::create_tables(tx)).unwrap();
```

Similar changes could be applied below too